package/package.sh

```diff
@@ -110,6 +115,12 @@ EOF
 
 function _build_graph {
     pushd ${build_dir}
+    if [ "x"$enable_breakpad == "xON" ]; then
+        if [ "x"$build_type != "xDebug" ] && [ "x"$build_type != "xRelWithDebInfo" ]; then
+            echo 'Breakpad need debug info.'
```

Comment:
It's better to check this condition in `BreakpadConfig.cmake` to avoid user to misuse the `CMAKE_BUILD_TYPE` and `ENABLE_BREAKPAD`.

Reply:
We can't turn on or turn off breakpad by `cmake_build_type`.